apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/create-page/index.tsx

```diff
@@ -43,13 +44,18 @@ export default function CreatePage( { postType }: Props ) {
 		postType.slug
 	);
 
+	const trackEvent = () => {
+		recordTracksEvent( `calypso_editor_sidebar_${ postType.slug }_add` );
```

Comment:
I'm not sure whether it's best to to track the `_add` event as two seperate events (one for posts and one for pages) or as one event. The advantage of a single event is it makes it easier to determine how often users are creating _anything_ using the sidebar, and I _think_ that's what we're more interested in.

We can still track the post type just in case with an event param:
```suggestion
		recordTracksEvent( 'calypso_editor_sidebar_add', { post_type: postType.slug } );
```

Reply:
What about `calypso_editor_sidebar_item_add`?